### ISSUE TITLE
Something went wrong when submitting a survey in batch

### ISSUE THREAD

**Issue by mzagaja** [MAPC/myschoolcommute2#120]
When a batch survey is submitted I get a "something went wrong" message.

**Comment by mzagaja** [comment]
This appears to be the result of rails strong parameters:
```
Started POST "/survey_responses" for 127.0.0.1 at 2017-07-25 16:59:12 -0400
Processing by SurveyResponsesController#create as HTML
  Parameters: {"utf8"=>"✓", "authenticity_token"=>"gr2tdsAxga1ScfLnO6+NZR0izh4VJnzZzfpxQO8ooBRPxu7eJc0QJvErHzwECV6jWhW8dE7Ru7gVrnGjgTtpkg==", "-search"=>"", "survey_response"=>{"grade_1"=>"k", "to_school_1"=>"w", "from_school_1"=>"w", "nr_vehicles"=>"1", "nr_licenses"=>"1", "survey_id"=>"30753"}, "commit"=>"Submit Survey"}
Unpermitted parameters: utf8, authenticity_token, -search, commit
   (0.2ms)  BEGIN
   (0.1ms)  ROLLBACK
Redirected to http://127.0.0.1:3000/
Completed 302 Found in 4ms (ActiveRecord: 0.3ms)
```
Not sure why they are not permitted given that they seem to be allowed in the controller with the following code:

```
def survey_response_params
      keys = params[:survey_response].keys
      params.permit(survey_response: [keys]).fetch(:survey_response)
    end
```

**Comment by mzagaja** [comment]
Specifically enumerating all the parameters has resolved the above portion of the bug. However, the app is still not getting a "survey id" or lat/long over the wire from the front end as shown by the following rails log:

```
Started POST "/survey_responses" for 127.0.0.1 at 2017-08-02 15:33:25 -0400
Processing by SurveyResponsesController#create as HTML
  Parameters: {"utf8"=>"✓", "authenticity_token"=>"wMtXMRfRXwWL7BX7DbWQwMYH+cYnFnbDae9a4nb1huEtHeV9oMlY1/wz3cEk8cKuTDm00gqMcLAraItWKpTjhw==", "-search"=>"", "survey_response"=>{"grade_1"=>"k", "to_school_1"=>"w", "from_school_1"=>"w", "nr_vehicles"=>"1", "nr_licenses"=>"1", "survey_id"=>""}, "commit"=>"Submit Survey"}
```

**Comment by allthesignals** [comment]
See the `-search` key - the way the input is configured in the React modules is affecting the naming of the parameters. Related to #130 #127 I think.

**Comment by mzagaja** [comment]
Resolved in #132
